js/explaingit.js

```diff
@@ -5,8 +5,9 @@ define(['historyview', 'controlbox', 'd3'], function (HistoryView, ControlBox, d
         reset,
         explainGit;
 
-    open = function (args) {
-        var name = prefix + args.name,
+    open = function (_args) {
+        var args = Object.create(_args),
```

Comment:
Moved `Object.create()` into `explainGit.open()`.

Renaming the parameter was harder to screw up then changing all internal references.